styles/colors-base.less

```diff
@@ -234,14 +235,18 @@
 @agate-item-bg-color:                     transparent;
 @agate-item-border-color:                 transparent;
 @agate-item-color:                        @agate-foreground;
+@agate-item-disabled-border-color:        @agate-item-border-color;
+@agate-item-disabled-color:               @agate-item-color;
+@agate-item-disabled-opacity:             @agate-disabled-opacity;
 @agate-item-label-color:                  @agate-item-color;
+@agate-item-label-focus-selected-color:   @agate-spotlight-color;
 @agate-item-focus-bg-color:               @agate-background-focus;
+@agate-item-focus-bg-image:               none;
 @agate-item-focus-border-color:           transparent;
 @agate-item-focus-color:                  @agate-foreground-focus;
 @agate-item-selected-bg-color:            @agate-item-bg-color;
 @agate-item-selected-border-color:        @agate-item-border-color;
 @agate-item-selected-color:               @agate-item-color;
```

Comment:
I think we should remove this. It depend on Gallium.
Should the font color change when selecting in Gallium?

when I set `disabled` and set `select`, the font color is changed. it is not needed on Silicon.

Reply:
This seems to be solved by changing `@agate-item-disabled-color: fade(@agate-foreground, 40%);` to `@agate-item-disabled-color: @agate-foreground;`. Now the font color remains the same when disabled and selected